pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -74,20 +60,21 @@ var _ = Describe("Node-labeller ", func() {
 		mockQueue = testutils.NewMockWorkQueue(nlController.queue)
 
 		nlController.queue = mockQueue
-		addNode(newNode("testNode", nodeLabels, nodeAnnotations))
 	}
 
 	BeforeEach(func() {
-		stop = make(chan struct{})
+
 		ctrl = gomock.NewController(GinkgoT())
 
+		node := newNode("testNode")
+
+		mockQueue.ExpectAdds(1)
```

Comment:
can you explain why `addNode()` is not necessary? To me it looks like it hides an implementation detail that not many will care to see as part of the test flow.

Reply:
I believe this can simply be inlined as it doesn't seem to be ever reused.